docs/concepts/05_credentials/05_verification.md

```diff
@@ -3,68 +3,97 @@ id: verification
 title: Verification
 ---
 
-KILT allows a Verifier to check if the information in a credential presented by a Claimer is correct and valid.
+KILT lets a Verifier check if the information in a credential presented by a Claimer is correct and valid.
 With the presentation of the credential, the Claimer also presents evidence that a third party (i.e., an Attester) ensured the correctness of the Claimer’s attributes.
 
-The Verifier trusts this third party either because they trust their reputation directly or they trust a delegation structure that this Attester is part of (e.g., a State department issuing driving licenses).
+The Verifier trusts this third party either because they trust their reputation directly or they trust a delegation structure that this Attester is part of.
+For example, a State department issuing driving licenses.
 
 For the verification process:
 
-- The Claimer needs their credential and the private key associated with their identifier
-- The Verifier needs the identifier of the trusted Attester
+-   The Claimer needs their credential and the private key associated with their identifier
+-   The Verifier needs the identifier of the trusted Attester
 
 During the verification process the Claimer wants to prove three things to the Verifier:
 
```

Comment:
Should we mention something here about the entity attesting?

Like the claimer wants to show that his information is certified by a trustworthy entity.

Reply:
@kilted-andres Could you explain a little more as I've not heard of this. This is like some way of showing a "badge" or some such?